module-2/lab-matplotlib-seaborn/your-code/challenge-3.ipynb

```diff
@@ -31,9 +31,17 @@
   },
```

Comment:
- The same as in challenge 2, try to include only numeric variables in describe method. Including IDs in this matrix is useless and can cause some confusion. 
- Once you have identified your categorical variables in your dataframe, set the type as category. You can take advantage of some useful methods. 
- Really good one!! This is the best to plot the histogram by age ranges instead of absolute values. 
```Python
bins=range(0,int(max(titanic['Age']))+1,10),ax=ax[1]
```
- This one "Use seaborn to show the distribution of the column Age" , 
"If you use titanic['Age'] without specifying that we want the values, it will deliver an warning message that  even being a pandas series, working fine right now, it wont be taken as a series due to its index.Remember this for the futuuuuureeee or Fry wont be able to get out from the cryogenic capsule!" This doesn't happen to me without using .values!! May we take a look on your notebook? 
- In this question: you can also run a seaborn jointplot. The scatter in the middle, kde of variables considered in each axis. 
- Regarding correlation matrix palette color, it's common use just one color but degraded depending on the level of correlation. It's more visual.